src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -2,29 +2,16 @@
 
 """Analyze ppsdpll log messages"""
 
-from .analyzer import Analyzer
+from .analyzer import TimeErrorAnalyzer
 
-class PhaseOffsetTimeErrorAnalyzer(Analyzer):
-    """Analyze DPLL phase offset time error"""
-    id_ = 'ppsdpll/phase-offset-time-error'
-    parser = 'dpll/phase-offset'
+class PpsDpllAnalyzer(TimeErrorAnalyzer):
+    """Analyze DPLL time error"""
+    id_ = 'ppsdpll/time-error'
+    parser = id_
     def __init__(self, config):
         super().__init__(config)
-        # required system time output accuracy
-        accuracy = config.requirement('time-error-in-locked-mode/ns')
-        # exclusive upper bound on absolute time error for any sample
-        self._unacceptable = accuracy
+        self._lockid = {3}
     def prepare(self, rows):
         return super().prepare([
-            r._replace(phaseoffset=float(r.phaseoffset)) for r in rows
+            r._replace(terror=float(r.terror)) for r in rows
```

Comment:
Should this modify the incoming data? Might we want to use the same data on other Analysers?

Reply:
NVM `_replace` returns a new object

Reply:
Is there a method on the row that can be used for this that does not have the underscore? convention is that a preceding underscore denotes a function intended to be treated as private.